config/prow_common.mk

```diff
@@ -100,10 +100,14 @@ update-almost-all-cluster-deployments:: confirm-master
 	$(SET_CONTEXT)
 	@for f in $(filter-out $(wildcard $(PROW_DEPLOYS)/*boskos*.yaml),$(wildcard $(PROW_DEPLOYS)/*.yaml)); do kubectl apply -f $${f}; done
 	$(UNSET_CONTEXT)
+
+update-almost-all-build-cluster-deployments:: confirm-master
```

Comment:
We are adding too many commands in this Makefile I think... Most of the commands are actually not needed, as all the deployments are idempotent.

Reply:
I'll take a look

Reply:
Removed some of the targets